delete-old-child-nodes/functions/index.js

```diff
@@ -0,0 +1,21 @@
+const functions = require('firebase-functions');
+const admin = require('firebase-admin');
+admin.initializeApp(functions.config().firebase);
+
+
+exports.deleteOldItems = functions.database.ref('/path/to/items/{pushId}')
+.onWrite(event => {
+  var ref = event.data.ref.parent; // reference to the items
+  var now = Date.now();
+  var cutoff = now - 2 * 60 * 60 * 1000;
+  var oldItemsQuery = ref.orderByChild('timestamp').endAt(cutoff).limitToLast(1);
+  return oldItemsQuery.once('value', function(snapshot) {
```

Comment:
for `.once` let's use the promise style instead: `.once('value').then(snapshot => {...`